cli/replicaset/cartridge.go

```diff
@@ -857,7 +872,7 @@ type cartridgeJoinServersOpts struct {
 type cartridgeEditReplicasetsOpts struct {
 	UUID             *string                    `msgpack:"uuid,omitempty"`
 	Alias            *string                    `msgpack:"alias,omitempty"`
-	Roles            []string                   `msgpack:"roles,omitempty"`
+	Roles            []string                   `msgpack:"roles"`
```

Comment:
Why the change is required?

Reply:
In case we delete last role we send empty []Roles and `omitempty` ignores it. As a result we cannot update roles.

Reply:
Won't roles be removed for other command that call the eval then?

Reply:
Sorry, I wasn't right. Backed `omitempty` as it was and everything works in expected way.

Reply:
No, it won't. If we do not pass `roles` field it will be initialized with a nil value which is `nil`. So it won't erase existing roles.